doc/sphinx/conf.py

```diff
@@ -66,11 +73,38 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
+# Add extra cases here to support more formats
+
+SUPPORTED_FORMATS = ["html", "latex"]
+
+def readbin(fname):
+    try:
+        with open(fname, mode="rb") as f:
+            return f.read()
+    except FileNotFoundError:
+        return None
+
+def copy_formatspecific_files(app):
+    ext = ".{}.rst".format(app.builder.name)
+    for fname in sorted(os.listdir(app.srcdir)):
+        if fname.endswith(ext):
+            src = os.path.join(app.srcdir, fname)
+            dst = os.path.join(app.srcdir, fname[:-len(ext)] + ".rst")
+            if readbin(src) == readbin(dst):
+                app.info("Skipping {}: {} is up to date".format(src, dst))
+            else:
+                app.info("Copying {} to {}".format(src, dst))
+                copyfile(src, dst)
+
+def setup(app):
+    app.connect('builder-inited', copy_formatspecific_files)
+
 # The master toctree document.
-master_doc = 'index'
+# We create this file in `copy_master_doc` above.
+master_doc = "index"
 
 # General information about the project.
-project = 'Coq'
+project = 'The Coq Reference Manual'
 copyright = '1999-2018, Inria'
```

Comment:
Can you tell me about what is the use of this copyright line? It seems it isn't used because the build uses timestamps instead. That's not a good idea because when building on Nix, timestamps are reset to a default value (1970). This doesn't really make sense from a copyright law point of view either. So it would be best if this line was actually used.

Reply:
I think the line is used, actually: at the bottom of the refman that's current online I see "© Copyright 1999-2018, Inria.".  Where is it not using it?

Reply:
This line at the bottom was generated using the timestamp. See https://coq.inria.fr/distrib/V8.8.0/refman/ which was built on NixOS.

Reply:
What sorcery is this? ^^ OK, I'll check.

Reply:
I tried `faketime -f "-10y" make sphinx-html` and changing to 2016 instead of 2018.  In both cases the copyright line didn't change.  Could there be something nix-specific going on?

Reply:
Let's continue this discussion in #7378.